main.js

```diff
@@ -0,0 +1,60 @@
+const key = '75f4785546ba2c0a068da88d46e4db01';
+let forecast = [];
+
+// Adds functoinality to search button
+document.querySelector('.search').addEventListener('click', function(){
+  var query = document.querySelector('#search-city').value;
+  fetchData(query);
+  document.querySelector('.form-control').value = '';
+
+});
+
+// Retrieves weather data from openweathermap
+const fetchData = function(query) {
+let url = 'https://api.openweathermap.org/data/2.5/forecast?q=' + query + '&appid=' + key;
+
+fetch(url, {
+  method: 'GET',
+  dataType: 'json'
+})
+.then(weatherData => weatherData.json()).then(weatherData => {getWeatherData(weatherData)})
```

Comment:
Suggestion: spacing could be improved to make this more readable

fetch(url, {
  method: 'GET',
  dataType: 'json'
})
.then(weatherData => weatherData.json())
.then(weatherData => getWeatherData(weatherData))